src/graphicswin.cpp

```diff
@@ -6,6 +6,8 @@
 //-----------------------------------------------------------------------------
 #include "solvespace.h"
 
+#include <iostream>
```

Comment:
What?!?! No!

Reply:
Whoops, thank you for the catch!  Looks like I left that in after some initial debugging.